test/Microsoft.ML.Tests/Microsoft.ML.Tests.csproj

```diff
@@ -21,5 +21,10 @@
     <NativeAssemblyReference Include="CpuMathNative" />
     <NativeAssemblyReference Include="FastTreeNative" />
     <NativeAssemblyReference Include="LdaNative" />
+	<NativeAssemblyReference Include="SymSgdNative" />
+  </ItemGroup>
+  
+  <ItemGroup>
+    <PackageReference Include="MlNetMklDeps" Version="0.0.0.1" />
```

Comment:
`0.0.0.1`  - this version number should go in 1 place - in the https://github.com/dotnet/machinelearning/blob/master/build/Dependencies.props file.

Reply:
Done.